applications/FluidDynamicsApplication/custom_processes/distance_modification_process.cpp

```diff
@@ -359,14 +357,14 @@ void DistanceModificationProcess::RecoverDeactivationPreviousState(){
             auto it_node = mrModelPart.NodesBegin() + i_node;
             if (it_node->GetValue(EMBEDDED_IS_ACTIVE) == 0){
                 for (int i_var = 0; i_var < static_cast<int>(mDoubleVariablesList.size()); i_var++){
```

Comment:
Why do you need the cast in here?

Reply:
I dont need it actually, I thought I did since it was inside the OMP loop.

Reply:
The parallel loop is the outer one. You can use an `unsigned int` iterator in here. Actually, you can even use an auto loop.